src/CustomElements/v1/README.md

```diff
@@ -0,0 +1,139 @@
+# Custom Elements v1
+
+## Status
+
+The polyfill should be mostly feature complete now. It supports defining
+custom elements, the custom element reactions, upgrades, and integrates with
+native Shadow DOM v1, and native and polyfilled HTML Imports.
```

Comment:
This sentence stumbles a bit. Consider:

It supports defining custom elements, lifecycle reactions (e.g. attach, detach), and upgrading existing elements. It integrates with native Shadow DOM v1, as well as both native and polyfilled HTML Imports.